.github/workflows/antsibull-build.yml

```diff
@@ -61,7 +61,7 @@ jobs:
           python3 -m pip install poetry ansible-core
           poetry install
           poetry update
-          ansible-galaxy collection install 'git+https://github.com/ansible-collections/community.general.git'
+          ansible-galaxy install -r requirements.yml
```

Comment:
This must still install from `git+https`. CI is already flaky enough thanks to Galaxy...

Reply:
Yeah that's why I added `type: git` in the requirements.yml file.

Ryt now we use `build-release` role in 1) GitHub Actions job and 2) while building Ansible release tarball so thought of having the galaxy requirement file in antsibull root and use the same for both.